### PR TITLE
Updated Artist Show Page Styling to Leverage CSS Grid

### PR DESCRIPTION
**Pull request by kneyzberg**
### Problem 
On Mobile we want to present the artist show page information in a different order than when it's on desktop/larger screens. 

### Solution 
Use CSS grid instead of  leverage grid-areas that you can reorder at different breakpoints. 

- updated logic to include profile photo if it's present.
- updated CSS on artist show page to use css grid and leverage the ordering of sections on mobile.

### CODE REVIEW

**Comment by kneyzberg** [comment]
I feel like the gutters are really big when the screen is big, but not sure how to fix that and keep the images "squarer". 
Screen Shot 2024-01-25 at 10 10 11 PM

**Review by bunnymatic — APPROVED** [review]
super first pass.  I think about your comment on the gutters, maybe a better way would be to make the full page have a max width so when the screen is 5000px wide, we still render the content next to each other.

but the beauty is we're getting into the details which means the big stuff is getting better.

also, I thiink rightly so, Trish is way more worried about mobile that someone's huge monitor.

**Comment by bunnymatic** [comment]
If you're feeling brave after you merge, deploy it!

File: frontend/stylesheets/artist_page.scss [hunk 1]
```diff
@@ -3,57 +3,43 @@
...
-  .artist-page--artwork--title {
-    border-bottom: 1px dotted grey;
-  }
+section.artist-page--social {
+  grid-area: social;
+}
 
+section.artist-page--artwork {
+  grid-area: artwork;
+  margin-top: 10px;
```
[the remark this answers is not in the document]
Comment (kneyzberg) [reply frontend/stylesheets/artist_page.scss:23]: Ah right, copy that

**Pull request merged**